sql/core/src/main/scala/org/apache/spark/sql/streaming/progress.scala

```diff
@@ -254,3 +259,10 @@ class SinkProgress protected[sql](
     }
   }
 }
+
+private[sql] object SinkProgress {
+  val DEFAULT_NUM_OUTPUT_ROWS: Long = -1L
```

Comment:
Does it result in sink progress output with "numOutputRows = -1" ? Maybe add numOutputRows to the output only if the value is not default.

Reply:
I will implement this for continuous streaming and then only legacy sinks would output -1. I didn't wanted to change the API too often.